src/components/WorkshopPage/WorkshopPage.js

```diff
@@ -277,7 +278,12 @@ function WorkshopPage() {
 					<Typography variant='h4' component='h1' className={classes.title}>
 						Workshops
 					</Typography>
-					{workshopCards}
+					{!workshopsAvailable ?
+						<ComingSoon alignment='left' /> :
+						<div>
```

Comment:
Semantically, I don't think this div around the `{ workshopCards }` is necessary, so I think we can delete that